lib/Runtime/Language/ModuleNamespace.cpp

```diff
@@ -170,6 +170,42 @@ namespace Js
         return HasProperty(propertyId);
     }
 
+    BOOL ModuleNamespace::IsConfigurable(PropertyId propertyId)
+    {
+        SimpleDictionaryPropertyDescriptor<BigPropertyIndex> propertyDescriptor;
+        const Js::PropertyRecord* propertyRecord = GetScriptContext()->GetThreadContext()->GetPropertyName(propertyId);
+        if (propertyMap != nullptr && propertyMap->TryGetValue(propertyRecord, &propertyDescriptor))
```

Comment:
This still doesn't match the order of what GetPropertyQuery does, which check if it is symbol and only then  we will pass it on to DynamicObject?

We should keep these logic in sync.